src/less/progress-stepper/base/progress-stepper.less

```diff
@@ -36,13 +35,25 @@ hr.progress-stepper__separator {
     width: @progress-stepper-min-width;
 }
 
+.progress-stepper__text h1,
```

Comment:
Realistically, with the document structure we are trying to promote. I think this heading will only ever be h3 level, sometimes h4. For example, the page would be h1. The stepper as a whole would be preceded by an h2 or h3 (e.g. "Shipping Progress").

Reply:
Okay, _maaaaybe_ h2 if the page literally only contained a progress stepper :-)